render-ws-java-client/src/main/java/org/janelia/render/client/newsolver/solvers/ADDSolverClient.java

```diff
@@ -11,13 +11,14 @@
 import java.io.IOException;
 
 /**
+ * Alternating Domain Decomposition solver:
  * A solver that alternates between solving for a given block layout and a shifted one so that
  * all tiles are placed in the center of a block in one of the iterations.
  * See also: additive Schwarz domain decomposition.
  *
  * @author Michael Innerberger
  */
-public class AlternatingDomainDecompositionSolverClient {
+public class ADDSolverClient {
```

Comment:
I prefer the slightly longer descriptive name - just don't see much value in abbreviating this here.